api/src/main/java/org/openmrs/Field.java

```diff
@@ -14,31 +14,56 @@
 
 import org.codehaus.jackson.annotate.JsonIgnore;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
 /**
  * Field
  *
  * @version 1.0
  */
+@Entity
+@Table(name = "field")
 public class Field extends BaseChangeableOpenmrsMetadata {
 	
 	public static final long serialVersionUID = 4454L;
 	
 	// Fields
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "field_id_gen")
+	@SequenceGenerator(name = "field_id_gen", sequenceName = "field_field_id_seq")
+	@Column(name = "field_id", nullable = false, updatable = false)
```

Comment:
@alinmihaila99, just a minor change, could you not plainly use `GenerationType.IDENTITY` here?